openc3-cosmos-cmd-tlm-api/app/controllers/activity_controller.rb

```diff
@@ -26,6 +26,7 @@
 class ActivityController < ApplicationController
   def initialize
     super()
+    @not_found = 'not found'
```

Comment:
Constant strings should be Constants, not instance variables.   applies throughout.

Reply:
We do ALL_CAPS, so NOT_FOUND in this case